### ISSUE TITLE
Add Stylus and LESS?

### ISSUE THREAD

**Issue by celsomiranda** [csstools/sanitize.css#21]
How would feel about adding Stylus and LESS versions of  `sanitize.scss`?

I can make a PR with the current version of the SCSS file converted into those preprocessors if you want.

**Comment by celsomiranda** [comment]
Well, I've taken a closer look, and although I did somewhat build a Stylus version, Less doesn't support conditions in variables like you use to define `$form-element-min-height`.

So I guess I can only contribute with the Stylus version.

**Comment by sotayamashita** [comment]
@celsomiranda  I am creating stylus version here, https://github.com/sotayamashita/sanitize.stylus

**Comment by sotayamashita** [comment]
@celsomiranda  I think if you would like to other version, you should fork it and create project.

**Comment by jonathantneal** [comment]
I’m open to accepting other versions. It might be nice if we used PostCSS to generate the alternate versions from one master pristine copy.

**Comment by sotayamashita** [comment]
@jonathantneal I will send you pull request of stylus version

**Comment by sotayamashita** [comment]
> It might be nice if we used PostCSS to generate the alternate versions from one master pristine copy.

Sound good!! but postCSS generate stylus file from SASS or CSS?

**Comment by ai** [comment]
@sotayamashita we still need Stylus stringifier, but technically PostCSS can do it.

Right now you can manually generates Stylus string by PostCSS AST (technically it is stringifier).

**Comment by sotayamashita** [comment]
@ai Sounds nice. How about LESS ?

**Comment by ai** [comment]
@sotayamashita as I understand Less syntax can be reproduced by non-standard selectors. So you can build Less sources on top of CSS stringifier.

``` js
let mixin = postcss.rule({ selector: 'mixin()' });
let variable = postcss.decl({ prop: '@var', value: '1' });
```

**Comment by sotayamashita** [comment]
@ai Sorry not to replay. I appreciate your kindest reply.  I will try to create this feature with PostCSS

**Comment by jonathantneal** [comment]
@sotayamashita, we still don’t have a less version. Would you be interested in creating this or should we close this issue?

**Comment by sotayamashita** [comment]
@jonathantneal you have stylus version ?

**Comment by jonathantneal** [comment]
@sotayamashita, [yes](https://github.com/10up/sanitize.css/blob/master/sanitize.styl)?

**Comment by sotayamashita** [comment]
@jonathantneal I got it and I create it today !! :rocket:  After sending PR, please close this

**Comment by jonathantneal** [comment]
Hey @sotayamashita, wanted to get the new release out so I figured this part out, using codepen to verify the results. Released as part of [v3.1.1](https://github.com/10up/sanitize.css/tree/v3.1.1). Thanks for the suggestion all that time ago.
